CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.4.0)
+
+project(sparse)
+
+add_subdirectory(sparse)
```

Comment:
The issue is on this line according to the [CI logs](https://github.com/pydata/sparse/pull/487/checks?check_run_id=2821436520#step:4:134).